src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -36,7 +36,6 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.6'">
-    <Compile Remove="SelfInitializedFakes/**" />
     <Compile Remove="Recorders.cs" />
     <Compile Remove="Creation/IFakeOptionsForWrappers.cs" />
     <Compile Remove="Creation/IFakeOptionsForWrappers.of.T.cs" />
```

Comment:
Can the three previous lines not also be removed? I guess that would mean the whole `ItemGroup`.